### ISSUE TITLE
Suppress warning output when files are excluded

### ISSUE THREAD

**Issue by LordFckHelmchen** [rvben/rumdl#686]
### Problem or motivation

Currently, when running rumdl (great tool, btw!) with the `exclude` flag set in the config, a warning will be issued for each excluded file, e.g. 

```bash
  warning: MY_FILE.md ignored because of exclude pattern 'MY_FILE.md'. Use --no-exclude to override
```

This floods the output of tools like `prek` and diverts from actual errors. For example, I exclude several auto-generated files (like the CHANGELOG.md), and will always see this warning pop-up in prek & in CI.

### Proposed solution

It would be great if the warning can be suppressed or is set to a different log-level (DEBUG?) so it won't show up in "normal" output. Since this is not not really an issue with the checked files, but a conscious configuration choice and should not result in a warning.

### Alternatives considered

I currently exclude the files via my prek-config to "suppress" the warnings since the files will then never be passed to rumdl. But since there is the config. option and it is much more handy then the according config option in prek, it would be great if the warning could just be removed and one could simply use the "exclude" option.

### Feature category

CLI / configuration

### Example markdown

```markdown
Example config, with MY_FILE.md being in the repo root


[tool.rumdl]
exclude = ["MY_FILE.md"]
```

### Contribution

- [x] I would be willing to submit a PR to implement this feature

**Comment by rvben** [comment]
Fixed in v0.2.23. The exclude notice is now `--verbose`-only, so it no longer shows in default/`prek`/CI output. `--no-exclude` still overrides exclusion.

Closing as released.

**Comment by LordFckHelmchen** [comment]
@rvben thanks a lot for the quick action!
